common/rust-toolchain

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
If you are leaving this at _stable_, might as well remove the file. If you want to specify MSRV, do so in `Cargo.toml`.